frontend/database/00242_add_index_identity_login_log_time_identity.sql

```diff
@@ -0,0 +1,2 @@
+-- Add index to indetity_login_log table on time and id_identity column
+CREATE INDEX idx_loginlog_time_identity ON Identity_Login_Log (time, identity_id);
```

Comment:
nit:
```suggestion
CREATE INDEX idx_loginlog_time_identity ON Identity_Login_Log (time, identity_id);

```
A Github (de hecho ese es tema de git) le gusta tener una línea en blanco al final de los archivos, pero el linter no se queja si no la agregas en los archivos `.sql`. 

Podemos tener contentos a los dos si la agregamos siempre